netman/adapters/switches/juniper/base.py

```diff
@@ -669,6 +669,10 @@ def fill_interface_from_node(self, interface, interface_node, config):
                 interface.trunk_vlans = vlans
             interface.trunk_native_vlan = value_of(self.custom_strategies.get_interface_trunk_native_vlan_id_node(interface_node), transformer=int)
             interface.shutdown = first(interface_node.xpath("disable")) is not None
+            if first(interface_node.xpath('ether-options/auto-negotiation')) is not None:
+                interface.auto_negotiation = True
+            elif first(interface_node.xpath('ether-options/no-auto-negotiation')) is not None:
+                interface.auto_negotiation = False
         return interface
```

Comment:
What if it is neither. What will the default be? If there is a default, should we just check for one of them?